pkgs/tools/virtualization/nixos-container/nixos-container.pl

```diff
@@ -25,7 +25,7 @@ sub showHelp {
        nixos-container create <container-name>
          [--nixos-path <path>]
          [--system-path <path>]
-         [--config <string>]
+         [--config <string>]  -- config without outer braces, for example 'services.nginx.enable = true;'
```

Comment:
This is already documented in `nixos/doc/manual/administration/imperative-containers.xml`.

Reply:
it's fine to duplicate it here. The hands-on `--help` is too scarce.

Reply:
I agree with @danbst here. This is a common misunderstanding (which usually seems like a bug to an end-user), so mentioning this here as well is fine IMHO.